### PR TITLE
Add “app container” to glossary

### PR DESCRIPTION
**Pull request by sftim**
Associated with #5993

### CODE REVIEW

**Comment by zparnold** [comment]
@sftim To have some context, is the concept of `init-container` documented as a glossary item?

**Comment by sftim** [comment]
There is a glossary entry for init container, and it reads:
> One or more initialization containers that must run to completion before any app containers run.
> 
> Initialization (init) containers are like regular app containers, with one difference: init containers must run to completion before any app containers can start. Init containers run in series: each init container must run to completion before the next init container begins.

**Comment by zparnold** [comment]
@sftim Now that workload definition has been merged, do you think it'd be good to link that as a reference here?

**Comment by sftim** [comment]
Expanded the definition a bit, including a cross-reference to workload (as @zparnold [suggested](https://github.com/kubernetes/website/pull/12611#issuecomment-464478830))

File: content/en/docs/reference/glossary/app-container.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+tags:
+- workload
+---
+ Application containers (or app containers) are the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}} that are started after any {{< glossary_tooltip text="init containers" term_id="init-container" >}} have completed.
+
+<!--more-->
+
+An init container lets you separate initialization details that are important for the overall 
+{{< glossary_tooltip text="workload" term_id="workload" >}}, and that don't need to keep running
+once the pod has started.
```
Comment (zparnold) [inline content/en/docs/reference/glossary/app-container.md]:
```suggestion
once the application container has started.
```

**Comment by zparnold** [comment]
@sftim One slight nit, then it's good to go.
/lgtm

**Comment by sftim** [comment]
@zparnold Could you review this once more?

**Pull request merged**